platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCPeopleProfileConnectionStatusTestIT.java

```diff
@@ -85,37 +83,205 @@ public void test01_CheckConnectionStatusBetweenTwoUsers() throws Exception {
     addUsers.addUser(username1, password, email1, username1, username1);
     addUsers.addUser(username2, password, email2, username2, username2);
     manageLogInOut.signIn(username1, password);
-
     info("goto user2's profile");
     homePagePlatform.goToConnections();
     $(ELEMENT_ALL_CONNECTIONS_TAB).click();
     connectionsManagement.searchPeople(username2, null, null, null);
     $(byXpath(ELEMENT_CONNECTION_USER_NAME.replace("${user}", username2))).click();
-
-    $(ELEMENT_UIMINICONNECTIONS_PORLET_CONNECT_STATUS).click();
-    $(ELEMENT_UIMINICONNECTIONS_PORLET_CANCEL_STATUS).waitUntil(visible, Configuration.timeout);
-
+    $(ELEMENT_ADD_CONNECT_PROFIL_STATUS).click();
+    $(ELEMENT_CANCEL_CONNECT_PROFILE_STATUS).waitUntil(visible,Configuration.timeout);
```

Comment:
We need to check also that this button has text(cancel request).Same for other buttons.